src/components/authentication/UserList.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
```

Comment:
Robbie's code today has been merged here!
I did some simple styling to make this page looks nicer.
I think we should rename this page to 'UserManagement' (or something similar) after Robbie finishes this page.